lib/active_record/connection_adapters/sqlserver/transaction.rb

```diff
@@ -7,10 +7,6 @@ module ConnectionAdapters
     module SQLServerTransaction
       private
 
-      def sqlserver?
```

Comment:
Think you will need to keep this. Or maybe replace it with:
```
delegate :sqlserver?, to: connection
```

Reply:
Hi @aidanharan I added your suggestion with the `prefix: true` option to help readability as well as tracking where the method is coming from at the transaction level. 

I've tested this locally on our app, it seem to work properly, I did not get the chance to setup the test env from the contributor manual. I'll try to do this this week.

Let me know if there is anything else.